File: RAI/clase9/Abstract/JAVA/Main.java
package JAVA;

public class Main{
    public static void main(String[] args){
        Dog firulais = new Dog("firulais","Golden Terrier","Mucho");
        Cat juanito = new Cat("Juanito", "Angora", "negras");

        firulais.makeSound();
        juanito.makeSound();

        Animal algun = new Animal();
        algun.makeSound();
    }
}

File: RAI/clase9/Abstract/PYTHON/Main.py
from Animal import Animal
from Dog import Dog
from Cat import Cat

def main():
    firulais = Dog("Firulais", "Golden Terrier", "Mucho")
    juanito = Cat("Juanito", "Angora", "negras")

    firulais.makeSound()
    juanito.makeSound()

    ##algun = Animal()
    ##algun.makeSound()


if __name__ == "__main__":
    main()